test/test.js

```diff
@@ -119,4 +119,28 @@ describe('postcss-increase-specificity', function() {
 	it('should not change the descendant rules of @keyframes', function() {
 		return testPlugin('./test/fixtures/keyframes.css', './test/fixtures/keyframes.expected.css');
 	});
+
+	it('should support disabled blocks of rules', function() {
+		return testPlugin('./test/fixtures/disabled-block.css', './test/fixtures/disabled-block.expected.css');
+	});
+
+	it('should support disabled blocks of rules with comments in between', function() {
+		return testPlugin('./test/fixtures/disabled-block-extra-comments.css', './test/fixtures/disabled-block-extra-comments.expected.css');
+	});
+
+	it('should support multiple disabled blocks of rules', function() {
+		return testPlugin('./test/fixtures/multiple-disabled-blocks.css', './test/fixtures/multiple-disabled-blocks.expected.css');
+	});
+
+	it('should support an entirely disabled stylesheet', function() {
+		return testPlugin('./test/fixtures/disabled-stylesheet.css', './test/fixtures/disabled-stylesheet.expected.css');
+	});
+
+	it('should support a partially disabled stylesheet', function() {
+		return testPlugin('./test/fixtures/partially-disabled-stylesheet.css', './test/fixtures/partially-disabled-stylesheet.expected.css');
+	});
+
+	it('should support disabled blocks of rules with declarations in between', function() {
+		return testPlugin('./test/fixtures/disabled-block-extra-declaration.css', './test/fixtures/disabled-block-extra-declaration.expected.css');
```

Comment:
wdyt of this name to describe the situation better? `extraneous-declaration-outside-rule-between-disable-comment`